youtube_dl/extractor/vier.py

```diff
@@ -45,6 +49,24 @@ class VierIE(InfoExtractor):
         'only_matching': True,
     }]
 
+    def _real_initialize(self):
+        self._login()
```

Comment:
@dstftw: There is no reliable way to determine whether a video requires authentication based on the URL. Only after the webpage is downloaded can we check this. How do I best deal with this?

My current plan: search for a login form in the webpage, if there's one, log in and retry.